### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by b-onc**
Adds initial Contribution guide, as discussed in #ios

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Before submitting, please make sure you're finished with the PR (and all tests pass) and do not make changes until it's reviewed.
+
+## Implemented or changed a feature 🌈
+
+Guidelines on "Fixed a bug" part is applicable.
+
+## Prepared a new Release 🚀
+
+Please follow [release guide](https://github.com/GetStream/stream-chat-swift/blob/master/releasing.md) closely.
+Make sure to follow all QA test procedure for minor and major releases.
```
Comment (buh) [inline CONTRIBUTING.md]: The contributors can't make a release.

Comment (buh) [reply CONTRIBUTING.md]: I think it's good to describe how we are making a new release.

Comment (b-onc) [reply CONTRIBUTING.md]: Agreed, release can only be made by Stream employees. Where would you suggest I add this info? In another file? Or maybe I can change the text so it reads "If you're a Stream employee and want to make a release..." so it's clear this is intended for Stream employees.

On "How to make a release" it's explained in the `releasing.md` file, isn't it?

Comment (buh) [reply CONTRIBUTING.md]: I think it's obvious that Stream employees only can make a release. So no need to describe how to make a release for contributors.

Comment (b-onc) [reply CONTRIBUTING.md]: I've refactored it so it's clear contributors can't make a release, and this paragraph states how Stream employees make releases.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Implemented or changed a feature 🌈
+
+Guidelines on "Fixed a bug" part is applicable.
+
+## Our Release flow 🚀
+
+We make sure to follow all QA test procedure for minor and major releases. 
+
+We accumulate changes and release them in batches, unless high priority.
+We make sure to put staged changes (on master but not released in a version) as "Upcoming" in `CHANGELOG`.
```
Comment (buh) [inline CONTRIBUTING.md]: Can you add a link to `CHANGELOG` instead of code style?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Contributing to Stream Chat Swift SDK
+
+We're glad you want to contribute to the Stream team 🎉
+
+---
+
+_So you.._
```
Comment (buh) [inline CONTRIBUTING.md]: > So you..
Add the 3rd dot: it should be: `So you...`

And the last thing, can you update `CHANGELOG.md`:
- replace `##### Breaking changes` with `## Breaking changes`,
- then replace all `##` to `###`.

**Pull request merged**